samples/samples-csharp/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.1.419",
+    "version": "6.0.300",
```

Comment:
We should just get rid of the global.json here and let it bubble up to the top one then.